projects/schematics/src/add-spartacus/index.ts

```diff
@@ -144,20 +144,49 @@ function installPackageJsonDependencies(): Rule {
   };
 }
 
-function getStorefrontConfig(options: SpartacusOptions): string {
-  const baseUrlPart = `\n          baseUrl: '${options.baseUrl}',`;
-  const contextContent = !options.baseSite
-    ? ''
-    : `
+function parseCSV(
+  raw: string | undefined,
+  defaultValues: string[] = []
+): string {
+  if (!raw) {
+    return defaultValues.map((x) => `'${x}'`).join(', ');
+  }
+
+  return raw
+    .split(',')
+    .map((x) => `'${x}'`)
+    .join(', ');
+}
```

Comment:
I like it, but could we move it to utils? I think it has reusability potential.

Reply:
Sure, I agree.